src/less/tooltip/base/tooltip.less

```diff
@@ -162,6 +162,66 @@ span.tourtip {
     width: @icon-width;
 }
 
+.dialog-infotip[role="dialog"] {
+    .dialog-base();
+}
+
+.dialog-infotip__window {
+    .dialog-window();
+
+    padding-bottom: 15vh;
+    top: 15%;
+    width: calc(100% - 32px);
+
+    &::before {
+        border-radius: 7px;
+    }
+}
+
+.dialog-infotip__window:not(.dialog-infotip__window--full) {
+    flex-direction: row-reverse;
+    left: 50%;
+    max-width: 400px;
+    min-width: 200px;
+    top: 50%;
+    transform: translate(-50%, -25%);
+}
+
+.dialog-infotip__header {
+    .dialog-header-content(0);
+
+    align-items: flex-start;
+}
+
+.dialog-infotip__close {
+    background-color: @dialog-back-close-background-color;
+    border: 0;
+    border-top-right-radius: 7px;
```

Comment:
If we change the `margin` to be greater than `7px`, then this isn't needed. But then the `padding` will need to be reduced to compensate
So maybe `padding: 8px; margin: 8px;` works

Reply:
Thanks for the tip!